### DISCUSSION TITLE
Welcome to PyCBC-Tutorials Discussions!

### DISCUSSION THREAD

**Discussion by ahnitz** [gwastro/PyCBC-Tutorials#22] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nisaAMIR** [comment]
My name is Nisa Amir and I am wondering that how can I upload my own data in PyCBC? I have some data in .h5 file and I want to upload it and run tutorial 2 of visualizing the waveforms with this data. How can I do it?
